src/items/menu.js

```diff
@@ -138,8 +143,11 @@ export default class MenuPanelItem extends PanelItem {
           } else if (action === 'logOut') {
             logout(false);
           }
-        }
+        },
+        onclose: () => setTimeout(() => { this.state.isOpen = false; })
```

Comment:
Seems like one of my rules aren't triggering (or maybe I forgot to add it when transitioning to a packaged eslint ruleset...), but curly braces needs to have newlines.

```
=> {
  this.state.isOpen = false;
}
```